pkgs/by-name/na/navicat-premium/package.nix

```diff
@@ -0,0 +1,114 @@
+{
+  fetchurl,
+  appimageTools,
+  lib,
+  libGL,
+  glib,
+  glibc,
+  pango,
+  harfbuzz,
+  fontconfig,
+  libX11,
+  freetype,
+  e2fsprogs,
+  expat,
+  p11-kit,
+  libxcb,
+  libgpg-error,
+  stdenv,
+  cjson,
+  libxcrypt-legacy,
+  curl,
+  makeWrapper,
+  autoPatchelfHook,
+  libxkbcommon,
+  libselinux,
+}:
+let
+  pname = "navicat-premium";
+  version = "17.1.3";
+  src = fetchurl {
+    url = "https://dn.navicat.com/download/navicat17-premium-en-x86_64.AppImage";
```

Comment:
URL does not contain full version = not suitable for nixpkgs. Please archive it at https://web.archive.org/ and use that link instead.

Reply:
Hmm.. not sure then. I'll try to figure it out later.

Reply:
I've uploaded the AppImage here: `https://fliegendewurst.eu/archive/navicat17.1.3-premium-en-x86_64.AppImage`. But I don't think it's really suitable for nixpkgs. Then again, there is precedent for fetching archived copies from own sources in e.g. #350557